front-end/src/components/modal/create/VideoSelect.js

```diff
@@ -0,0 +1,65 @@
+import '../../../styles/modalStyles.css';
+import '../../../styles/videoSelectStyles.css';
+import { AiFillFolder } from 'react-icons/ai';
+
+export const VideoSelect = ({ movePage, selectedVideo, setSelectedVideo, files, filteredFiles, setFilteredFiles, fileTimespans, videoTimespans }) => {
```

Comment:
Why are filteredFiles here and selectedFiles two separate variables? Can we just update the earlier one, or is there some reason? Should probably name for clarity due to similarity now

Reply:
And then theres selectedVideo, which isnt corresponding to selectedFiles but filteredFiles

Reply:
there's no selected files

selectedVideo is required to know which object to highlight
setSelectedVideo is required to set the video that is selected since this is the videoSelect class
files is required since we need to know what files exist to filter them
filteredFiles is required since it is used to decide what folders to show in the available folders column
setFilteredFiles is required since they are calculated when a video is selected

selectedVideo is related to filteredFiles since filtered files are the files that overlap with the selected video

It says files not selected files. This page does not handle selecting of the files, the files is all the files since we need to know what they are for filtering

Reply:
I meant just renaming

Reply:
change filteredFiles to videoSyncFiles